### PR TITLE
travis stuff

### PR DESCRIPTION
**Pull request by kendricktan**

### CODE REVIEW

File: contracts/Heiswap.sol [hunk 1]
```diff
@@ -285,7 +285,7 @@ contract Heiswap {
 
     // Gets all addresses in a Ring
     // Converting to Bytes32 cause web3.js has a bug that doesn't convert
-    // Big Num correctly....
+    // BigNum correctly....
```
Comment (adrianmcli) [inline contracts/Heiswap.sol]: lol

Comment (kendricktan) [reply contracts/Heiswap.sol]: It's true

Comment (adrianmcli) [reply contracts/Heiswap.sol]: @kendricktan is there an issue on Web3.js for it?

Comment (kendricktan) [reply contracts/Heiswap.sol]: @adrianmcli it only occurs on some big integers (I.e < JS native 32 bit integers)

**Pull request merged**